cmd/configure.go

```diff
@@ -299,10 +294,10 @@ func readLegacyConfig() (map[string]any, error) {
 	f, err := os.Open(legacyCfgFile)
 	//lint:ignore SA5001 setting defer before the error check,
 	// because the function will return immediately with the error
```

Comment:
Do we need to remove these comments with moving the defer below the error check?